README.md

```diff
@@ -1,5 +1,8 @@
 # RETURN OF THE FRUSTRATING BUSMALL:
```

Comment:
pro tip: take out "frustrating". Every time you look at it, it'll remind you that you're frustrated. You need to be reminding yourself that you're a competent developer who can do this stuff.